player/command.c

```diff
@@ -604,7 +604,7 @@ static int property_time(int action, void *arg, double time)
     if (time == MP_NOPTS_VALUE)
         return M_PROPERTY_UNAVAILABLE;
 
-    const struct m_option time_type = {.type = CONF_TYPE_TIME};
+    const struct m_option time_type = {.type = CONF_TYPE_REL_TIME};
```

Comment:
this breaks `playback-time` property used by the osc

Reply:
Hm, that's actually far from the only thing that broke, I'd rather just drop the removing OPT_TIME for now

Reply:
What?

Reply:
I removed the top two commits that make a poor attempt at removing the "old" time option code, because there was more left using it than I thought there was and only kludged something together too quickly.
It's definitely possible to remove as I originally said, but it'll require more code changes than just removing a few lines and I don't want to rush it.

Reply:
re-added these commits with an extra one that properly converts property_time to rel_time, so the last two users of time options are properly transformed separately, but I'm not convinced it's worth removing the old code if it costs that much rework - I'm sure I've made more bugs than the cleanup is worth, I didn't realize there were two reasonably big users left of the option back when I suggested I could remove it.
Leaving the commits here for opinions but I'll be more than glad to remove the top 3 commits and focus on ironing ab-loop options properly.